oozie-to-airflow/utils/el_utils.py

```diff
@@ -11,13 +11,16 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
The changes to el_utils seem independent of the changes to email mapper. Can we split this into separate commits and likely separate PRs?